src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/button.scss

```diff
@@ -2,45 +2,58 @@
 
 $itemBackgroundColor: $alabaster;
 $itemTextColor: $blueZodiac;
+$itemTextActiveColor: $shakespeare;
 $itemTextColorDisabled: $silver;
+$itemActiveBackgroundColor: $wildSand;
 
-$itemBorder: 1px $wildSand solid;
+.button-container {
+    position: relative;
+    display: inline-block;
+}
 
-.item {
-    padding: 0 30px;
+.button {
+    position: relative;
+    padding: 0 20px;
     border: none;
-    outline: none;
     height: 60px;
     background: $itemBackgroundColor;
     color: $itemTextColor;
-    font-size: 14px;
+    font-size: 12px;
     cursor: pointer;
 
     & > * {
         /* IE11 will emulate 3D effect while clicking button without this */
```

Comment:
Out of scope of this PR. And since this happens as long as you hold the click, I think `while` is more appropriate 😃